src/components/Collections/CollectionOptions/index.tsx

```diff
@@ -232,30 +234,38 @@ const CollectionOptions = (props: CollectionOptionsProps) => {
     };
 
     return (
-        <OverflowMenu
-            ariaControls={'collection-options'}
-            triggerButtonIcon={<MoreHoriz />}
-            triggerButtonProps={{
-                sx: {
-                    background: (theme) => theme.palette.fill.dark,
-                },
-            }}>
-            {collectionSummaryType === CollectionSummaryType.trash ? (
-                <TrashCollectionOption
-                    handleCollectionAction={handleCollectionAction}
-                />
-            ) : collectionSummaryType ===
-              CollectionSummaryType.incomingShare ? (
-                <SharedCollectionOption
-                    handleCollectionAction={handleCollectionAction}
-                />
-            ) : (
-                <AlbumCollectionOption
-                    IsArchived={IsArchived(activeCollection)}
-                    handleCollectionAction={handleCollectionAction}
-                />
+        <Box sx={{ display: 'inline-flex', gap: '16px' }}>
```

Comment:
you can use `FlexWrapper`  , instead of creating a Box

Reply:
`FlexWrapper`'s "width: 100%" property is causing the header to shrink.
`HorizontalFlex` is working fine though.